src/mqt/qecc/circuit_compilation/code_switching_compiler.py

```
@@ -0,0 +1,234 @@
+
+        Parameters
+        ----------
+        control_node : str
+            Node representing the control qubit's CNOT operation.
+        target_node : str
+            Node representing the target qubit's CNOT operation.
+        """
+        self.add_infinite_edge(control_node, target_node, bidirectional=not (one_way_transversal_cnot))
+
+    def build_from_qiskit(
+        self, circuit: QuantumCircuit, one_way_transversal_cnot: bool = False, code_bias: bool = False
```

Ruff (0.14.3)

Boolean-typed positional argument in function definition

(FBT001)